vec_crc32.c

```diff
@@ -64,6 +64,8 @@ static unsigned int crc32_align(unsigned int crc, unsigned char *p,
 static unsigned int __attribute__ ((aligned (32)))
 __crc32_vpmsum(unsigned int crc, void* p, unsigned long len);
 
+unsigned int crc32_vpmsum(unsigned int crc, unsigned char *p,
+                unsigned long len);
```

Comment:
Was this really required to prevent a compile warning/failure? The function itself is defined 4 lines later.

Also the CRC32_FUNCTION defines the function name.

I've packed up the rest of the changes into https://github.com/racardoso/crc32-vpmsum/pull/4. When @racardoso merges that it gets added to the pr #4 here. Saves overlapping PRs from ending up here.

Reply:
Actually I don't think this is necessary. Will close the pull request.